incuna_auth/middleware/login_required.py

```diff
@@ -57,6 +57,9 @@ def __init__(self, check=True):
         if check:
             check_request_has_user()
 
+    def get_unauthorised_redirect_url(self, request):
+        return settings.LOGIN_REDIRECT_URL
```

Comment:
This does mean that a class overriding `get_unauthorised_redirect_url()` has an attribute called `unauthorised_redirect_url` that happens to be wrong.  (The request parameter isn't used in this library, but it's used in projects that extend the library.)